README.md

```diff
@@ -57,21 +57,21 @@ By default, `uuid` can be depended on with:
 
 ```toml
 [dependencies]
-uuid = "0.7"
+uuid = "0.8.0"
 ```
 
 To activate various features, use syntax like:
 
 ```toml
 [dependencies]
-uuid = { version = "0.7", features = ["serde", "v4"] }
+uuid = { version = "0.8.0", features = ["serde", "v4"] }
```

Comment:
we can keep it at 0.8, that way we don't have to update it for every minor release

Reply:
Sure! It ended up this way from my _find previous version text and replace with new version_ workflow, but I’m happy to change this to `0.8`.